src/main/java/seedu/address/model/student/AssessmentStatistics.java

```diff
@@ -86,7 +90,7 @@ private Bin getBin(Score score) {
             }
         }
 
-        // Should not happen since a Score must be between the minimum and maximum value (inclusive)
+        // Should not happen since a Score must be between the minimum and maximum name (inclusive)
```

Comment:
```suggestion
        // Should not happen since a Score must be between the minimum and maximum value (inclusive)
```

I think you accidentally find-and-replaced this 😅